.gitignore

```diff
@@ -84,12 +99,13 @@ src/app/jni/output
 # Python build directory
 build/
 
-# MSVC build directories
-obj/
-bin/
-*.tlog
+# Qt
+*pro.user
+moc_*
+build-*
+bindings/qt/selftest/selftest
 
 # Temporary files
 *.swp
+*.bak
 *~
```

Comment:
In general yes, but there are always a couple of exceptions which differ for each project.